.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.8", "3.11", "3.12"]
+        python-version: ["3.11", "3.12"]
```

Comment:
Since we're dropping the tests for Python 3.8, and we can no longer guarantee compatibility with Quince, should we bump the version, add an entry to the changelog, and update the [compatibility table](https://github.com/eduNEXT/codejailservice?tab=readme-ov-file#compatibility)?

Reply:
This plugin follows the tutor versioning convention, so this version (v17.x.x) doesn't necessarily support quince, although no significant changes have been made for a while. This makes me question if this plugin should follow the same convention cause it's pretty stable, so the latest release could be used without concern.

Now that Maga mentioned the compatibility table, can we update it to reflect the current plugin state?

Reply:
Reminder that this is not a Tutor plugin, this is essentially an IDA, `tutor-contrib-codejail` is the one that follows the Tutor versioning scheme. The actual plugin uses the compatible code of codejailservice for each release.

We decided to change the versioning scheme in redwood (see: https://github.com/eduNEXT/codejailservice/issues/22). What should happen is that a new sumac branch is created, `tutor-contrib-codejail` builds using that branch until the release is ready and after that we tag `sumac.1`. I don't think the compatibility table in this repo is useful and I would rather remove it.

One thing that can be improved though, we should tag the other point-releases when they are ready.

Reply:
Oh, sorry. I edited my comment so it references the service instead.

I wasn't aware of the change in the versioning scheme. Should we remove the service releases to make it less confusing?